docs/installation.md

```diff
@@ -154,67 +155,65 @@ vacuum = true
 reload-mercy = 8
 ```
 
-**`/etc/uwsgi/apps-available/udata-worker.ini`**
-
-```inifile
-##
-# uWSGI configuration for data.example.com worker
-##
-
-[uwsgi]
-master = true
-plugin = python
-home = /srv/udata
-chdir = %(home)
-virtualenv = %(home)
-pythonpath = %(home)/bin
-socket = /tmp/udata-worker.sock
-chmod-socket = 666
-processes = 1
-uid = udata
-gid = www-data
-smart-attach-daemon = /tmp/celery-worker.pid %(home)/bin/celery -A udata.worker worker --pidfile=/tmp/celery-worker.pid
-exec-as-user-atexit = kill -TERM `cat /tmp/celery-worker.pid`
-```
+Then create the symlink to activate this configuration:
 
-**note:** You can handle tasks priorities by starting 3 workers
-with the following `smart-attach-daemon` commands:
-```
-/tmp/celery-worker.pid %(home)/bin/celery -A udata.worker -Q high worker
-/tmp/celery-worker.pid %(home)/bin/celery -A udata.worker -Q high,default worker
-/tmp/celery-worker.pid %(home)/bin/celery -A udata.worker -Q high,default,low worker
+```shell
+$ ln -s /etc/uwsgi/apps-{available,enabled}/udata-front.ini
 ```
 
+You can now create the systemd unit file for Celery:
 
-**`/etc/uwsgi/apps-available/udata-beat.ini`**
+**`/etc/systemd/system/celery.service`**
```

Comment:
`udata-celery.service`? To match our config.

Reply:
I tried to simply it a little bit to make it easier to read